src/KeyboardAccessoryNavigation.js

```diff
@@ -136,9 +142,27 @@ KeyboardAccessoryNavigation.propTypes = {
   nextButtonStyle: (View.propTypes||ViewPropTypes).style,
   doneButtonStyle: (View.propTypes||ViewPropTypes).style,
   doneButtonTitleStyle: Text.propTypes.style,
+  doneButtonHitslop: PropTypes.shape({
```

Comment:
As I see from react-native documentation hitSlop can be Rect or Number. So it would be better to change accepted propTypes accordingly.

Reply:
I agree. I commit by modifying that part.